content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -87,13 +87,22 @@ Instead of immediately awaiting the result of each operation when it's performed
 >
 > Refer to the 📖 [transactions guide](../../../guides/performance-and-optimization/prisma-client-transactions-guide#transaction-api) for more examples.
 
-### Interactive Transactions
+### Interactive Transactions (in Preview)
```

Comment:
Unrelated change to PR - should probably be merged into the epic branch so it does not show up here.